dsc/sqs.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import annotations
+
+import json
+import logging
+from typing import TYPE_CHECKING, Any
+
+from boto3 import client
+
+from dsc.exceptions import InvalidSQSMessageError
+
+if TYPE_CHECKING:
+    from collections.abc import Iterator, Mapping
+
+    from mypy_boto3_sqs.type_defs import (
+        EmptyResponseMetadataTypeDef,
+        MessageAttributeValueTypeDef,
+        MessageTypeDef,
+        SendMessageResultTypeDef,
+    )
+
+logger = logging.getLogger(__name__)
+
+
+class SQSClient:
```

Comment:
Commenting up here but applies to all `logger` calls in the client: Can you update the log calls to use f-string?